src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -351,8 +351,6 @@ private void ProcessTraceroute(string targetNameOrAddress)
             IPAddress hopAddress;
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-linux`
Test-Connection.TraceRoute.has a non-null value for Destination for reachable hosts
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 282
282:             $results.Destination | Should -Not -BeNullOrEmpty
```